apps/web/src/app/_components/search/search-filter.tsx

```diff
@@ -22,8 +24,24 @@ export default function SearchFilter() {
     },
   });
 
+  const searchParams = useSearchParams();
+  const router = useRouter();
+  const pathName = usePathname();
+
+  const createQueryString = useCallback(
+    (name: string, value: string) => {
+      const params = new URLSearchParams(searchParams);
+      params.set(name, value);
+      return params.toString();
+    },
+    [searchParams],
+  );
+
   function onSubmit(values: z.infer<typeof formSchema>) {
-    console.log(values);
+    // console.log(searchParams.get("search")); query the user search info
```

Comment:
Since we are going to be getting the search params in the specific roles / companies page and not the search filter in the future could we remove this comment from here? (Just a nitpick idrgaf either way LOL)